stories/Welcome.mdx

```diff
@@ -1,7 +1,138 @@
-import './styles.css';
+import './welcome-styles.css';
+import gridironLogo from './assets/gridironSurvivorLogo.svg';
+import appwriteLogo from './assets/appwriteLogo.svg';
+import blogRecorderLogo from './assets/blogRecorderLogo.svg';
+import frontendMentorLogo from './assets/frontendMentorLogo.svg';
+import gitkrakenLogo from './assets/gitkrakenLogo.svg';
+import pastelLogo from './assets/pastelLogo.svg';
+import Image from 'next/image';
 
-<h1>Welcome</h1>
+<html>
+  <body class="dark bg-background text-muted-foreground max-w-screen-xl mx-auto text-pretty flex flex-col space-y-36 pb-36">
+    <header class="header flex justify-center align-middle">
+      <a href="https://www.gridironsurvivor.com" target="_blank" rel="noreferrer" class="w-2/6 h-fit">
+        <Image class="w-full" src={gridironLogo} alt='Gridiron Survivor Football Logo' />
+      </a>
+    </header>
+    <section class="hero space-y-12">
+      <h1 class="pl-16">What is Gridiron Survivor?</h1>
+      <p class="text-xl text-pretty max-w-screen-lg mx-auto">
+        Gridiron Survivor is the ultimate pick'em league app. Track your picks, see how you stack up against your friends, and hope to be crowned champion! Developed by a talented team of junior developers using technologies such as Next.js, React, and Tailwind CSS, Gridiron Survivor is the perfect tool for any football fan looking to take their pick'em game to the next level.
```

Comment:
nit: I'm unsure I how I feel about the choice of using the word pick'em. Maybe also consider removing junior, or add that it's led by Shashi, a senior engineer. I don't know about tool, maybe application is a better choice. We could also link to the Documentation research pages for Next.js, React, and Tailwind.

Reply:
Great feedback all around. I never meant for this to be the final iteration, but figured it was a place to start! Could use some feedback from others on the verbiage as well!

Reply:
Altered the copy for this.